src/components/learner-credit-management/SubBudgetCard.jsx

```diff
@@ -93,59 +105,62 @@ const BaseSubBudgetCard = ({
     const subtitle = (
       <Stack direction="horizontal" gap={2.5}>
         <Badge variant={budgetLabel.badgeVariant}>{budgetLabel.status}</Badge>
-        {(budgetLabel.term && formattedDate) && (
+        {budgetLabel.term && formattedDate && (
           <span data-testid="budget-date">
             {budgetLabel.term} {formattedDate}
           </span>
         )}
       </Stack>
     );
 
+    const showActions = budgetLabel.status !== BUDGET_STATUSES.scheduled;
+    const showBottomMargin = !isBudgetActiveOrRetired;
+
     return (
       <Card.Header
         title={<BackgroundFetchingWrapper>{budgetType}</BackgroundFetchingWrapper>}
         subtitle={<BackgroundFetchingWrapper>{subtitle}</BackgroundFetchingWrapper>}
-        actions={
-          budgetLabel.status !== BUDGET_STATUSES.scheduled
-            ? renderActions(budgetId)
-            : undefined
-        }
-        className={classNames('align-items-center', {
-          'mb-4.5': budgetLabel.status !== BUDGET_STATUSES.active && budgetLabel.status !== BUDGET_STATUSES.expiring,
-        })}
+        actions={showActions ? renderActions(budgetId) : undefined}
+        className={classNames('align-items-center', { 'mb-4.5': showBottomMargin })}
       />
     );
   };
 
-  const renderCardSection = () => (
-    <Card.Section
-      title={<h4>Balance</h4>}
-      muted
-    >
-      <Col className="d-flex justify-content-start w-md-75">
-        <Col xs="6" md="auto" className="mb-3 mb-md-0 ml-n4.5">
-          <div className="small font-weight-bold">Available</div>
-          <span className="small">
-            {isFetchingBudgets ? <Skeleton /> : formatPrice(available)}
-          </span>
-        </Col>
-        {isAssignable && (
+  const renderCardSection = () => {
+    if (!isBudgetActiveOrRetired) {
+      return null;
+    }
+
+    return (
+      <Card.Section
+        title={<h4>Balance</h4>}
+        muted
+      >
+        <Col className="d-flex justify-content-start w-md-75">
+          <Col xs="6" md="auto" className="mb-3 mb-md-0 ml-n4.5">
+            <div className="small font-weight-bold">Available</div>
```

Comment:
Per the Figma mock, it appears the retired budget cards are only supposed to show the "Spent" amount; however, the current implementation includes other amounts beyond "Spent".

Similarly, the Figma mock doesn't seem to include the "Balance" title. Not sure the rationale for why the retired budget cards wouldn't have the "Balance" title, but that's what is currently mocked.

<img width="669" alt="image" src="https://github.com/openedx/frontend-app-admin-portal/assets/2828721/532c9fce-7a6a-47e3-80d9-c343edaad7d7">